fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/legacy/generated/A32NX_Interior_Misc.xml

```diff
@@ -608,10 +608,10 @@
                                 <CLOCKWISE_CONDITION>(L:XMLVAR_Baro#BARO_ID#_Mode) #MODE_STD# !=</CLOCKWISE_CONDITION>
                                 <ANTICLOCKWISE_CONDITION>(L:XMLVAR_Baro#BARO_ID#_Mode) #MODE_STD# !=</ANTICLOCKWISE_CONDITION>
                                 <PULL_CODE>
-                                    #MODE_STD# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode)
+                                    (L:XMLVAR_Baro#BARO_ID#_MODE) #MODE_STD# == if{ #MODE_BARO# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) } els{ (L:XMLVAR_Baro#BARO_ID#_Mode) ! (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) }
                                 </PULL_CODE>
                                 <PUSH_CODE>
-                                    (L:XMLVAR_Baro#BARO_ID#_Mode) #MODE_STD# == if{ #MODE_BARO# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) } els{ (L:XMLVAR_Baro#BARO_ID#_Mode) ! (&gt;L:XMLVAR_Baro#BARO_ID#_Mode) }
+                                    #MODE_STD# (&gt;L:XMLVAR_Baro#BARO_ID#_Mode)
```

Comment:
Is this even still used?

Reply:
No that's not being used. I will remove it right now.